Fix code samples in the README.md (#49)
wso2-ballerina/module-googlespreadsheet: gsheets4/Module.md

The `openSpreadsheetById` function retrieves the spreadsheet whose ID is specified in `spreadsheetId`.
```ballerina
//Open a spreadsheet.
var response = spreadsheetClient->openSpreadsheetById(spreadsheetId);
```

The response from `openSpreadsheetById` is a `Spreadsheet` object if the request was successful or a `error` on failure. The `match` operation can be used to handle the response if an error occurs.
```ballerina
string spreadsheetId = "1Ti2W5mGK4mq0_xh9Gl_zG_dK9cRvdduirsFgl6zZu7M";
var response = spreadsheetClient->openSpreadsheetById(spreadsheetId);
if (response is gsheets4:Spreadsheet) {

>     //If successful, returns the Spreadsheet object.

Reviewer: ```suggestion
    // If successful, returns a `Spreadsheet` object.
```

    io:println("Spreadsheet Details : ");

>     io:println(response);

Reviewer: Let's merge both to say 
```suggestion
    io:println("Spreadsheet Details: ", response);
```

} else {

>     //Unsuccessful attempts return a error.

Reviewer: ```suggestion
    // Unsuccessful attempts return an `error`.
```

>     io:println(response);

Reviewer: ```suggestion
    io:println("Error: ", response);
```

The `openSpreadsheetById` function retrieves the spreadsheet whose ID is specified in `spreadsheetId`.
```ballerina
//Open a spreadsheet.
var response = spreadsheetClient->openSpreadsheetById(spreadsheetId);
```

The response from `openSpreadsheetById` is a `Spreadsheet` object if the request was successful or an `error` if unsuccessful.

```ballerina
string spreadsheetId = "1Ti2W5mGK4mq0_xh9Gl_zG_dK9cRvdduirsFgl6zZu7M";
var response = spreadsheetClient->openSpreadsheetById(spreadsheetId);
if (response is gsheets4:Spreadsheet) {
    //If successful, print the Spreadsheet object.

>     io:println("Spreadsheet Details : ", response);

Reviewer: ```suggestion
    io:println("Spreadsheet Details: ", response);
```

} else {
    // If unsuccessful, print the error returned.
    io:println("Error: ", response);
}
```

The `getSheetByName` function retrieves a sheet with the given name from a `Spreadsheet` object. The `sheetName` represents the name of the sheet to be retrieved. It returns the `Sheet` object on success or an `error` if unsuccessful.
```ballerina
string sheetName = "Sheet1";
var response = spreadsheet.getSheetByName(sheetName);
if (response is gsheets4:Sheet) {

>    //If successful, print the Spreadsheet Details.

Reviewer: ```suggestion
   // If successful, print the Spreadsheet Details.
```

>    io:println("Sheet Details : ", response);

Reviewer: ```suggestion
   io:println("Sheet Details: ", response);
```

} else {
   // If unsuccessful, print the error returned.
   io:println("Error: ", response);
}
```

> The `setSheetValues` function sets the values for a range of cells. It returns `true` on success or an `error` if unsuccessful.

Reviewer: ```suggestion
The `setSheetValues` function sets the values for a range of cells. It returns the `boolean` status if successful or an `error` if an error occurred.
```

```ballerina
var response = spreadsheetClient->setSheetValues(spreadsheetId, sheetName, topLeftCell = topCell, bottomRightCell = bottomCell, values);
if (response is boolean) {

>    io:println("Sheet Values: ", response);

Reviewer: ```suggestion
   io:println("Status: ", response);
```
